BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -199,6 +199,15 @@ extern NSString *const BOXContentClientBackgroundTempFolder;
  **/ 
 - (void)setSharedLinkStorageDelegate:(id <BOXSharedLinkStorageProtocol>)delegate;
 
+/**
+ * Sets the access token delegate for an App Auth client.
+ *
+ * @param accessTokenDelegate  The delegate responsible for fetching new tokens when necessary.
+ * @param serverAuthUser       The user for which this client should be authenticated.
```

Comment:
Maybe update description:
"The user for which this client is authenticated" or "Currently authenticated user for this client"